### ISSUE TITLE
multiple mobx instances active

### ISSUE THREAD

**Issue by vporoshok** [mobxjs/mobx-angular#37]
After update to 1.6.0 we got error on tests:
```
WARN: '[mobx] Warning: there are multiple mobx instances active. This might lead to unexpected results. See https://github.com/mobxjs/mobx/issues/1082WARN: '[mobx] Warning: there are multiple mobx instances active. This might lead to unexpected results. See https://github.com/mobxjs/mobx/issues/1082
 for details.'
```
And test failed on autorun waiting. It's because npm install mobx twice: in project and inside mobx-angular
![selection_005](https://user-images.githubusercontent.com/5063912/29460728-0cff47f0-843a-11e7-8ee5-88cd24658304.png)
After manual deleting mobx from mobx-angular everything work fine.

**Comment by adamkleingit** [comment]
True, it's a bug in 1.6.0.
MobX should be peerDependency.
Will fix soon in 1.7.0

**Comment by adamkleingit** [comment]
Hi was released in 1.7.0 - can you verify?
